development/config/Doxyfile

```diff
@@ -1,4 +1,4 @@
-# Doxyfile 1.8.12
+# Doxyfile 1.8.11
```

Comment:
I actually think that there is no need to modify this file in this PR, it's about something different.